### ISSUE TITLE
Deprecated function call

### ISSUE THREAD

**Issue by kflak** [davidgranstrom/telescope-scdoc.nvim#8]
Hi,

Getting this these days:

```
vim.tbl_islist is deprecated, use vim.islist instead. :help deprecated
Feature will be removed in Nvim 0.12
```

Time to update those function calls, I suppose :-)

**Comment by davidgranstrom** [comment]
Hi, 

`vim.tbl_islist` is not used in the extension implementation, but perhaps it is used by Telescope itself? I noticed the following commits when updating telescope recently:

```
        0c12735 fixup: adapt to deprecations in Nvim 0.10 (#3113) (4 days ago)
        33f04c3 fixup: adapt to deprecations in Nvim 0.10 (#3112) (4 days ago)
        bbdbb75 fix: adapt to Nvim deprecations in 0.10 (#3109) (4 days ago)
```

Perhaps updating Telescope will solve this issue?

**Comment by kflak** [comment]
Hi,

Indeed a telescope thing! I updated now, and the error is gone...
